Os/Posix/Task.cpp

```diff
@@ -19,14 +19,18 @@
 
 typedef void* (*pthread_func_ptr)(void*);
 
-//#define DEBUG_PRINT(x,...) Fw::Logger::logMsg(x,##__VA_ARGS__);
-#define DEBUG_PRINT(x,...)
+void* pthread_entry_wrapper(void* arg) {
+    Os::Task::TaskRoutineWrapper *task = (Os::Task::TaskRoutineWrapper*) arg;
```

Comment:
Probably should add an assert that arg is not NULL, and that the members aren't NULL.

Reply:
I added an assert that routine was non-null. Null task arguments are okay.

Reply:
Oh I see what you mean now lol, I'll fix that